```
************* Module bot
bot.py:92:0: C0116: Missing function or method docstring (missing-function-docstring)
bot.py:94:8: C0103: Variable name "summonerInfo" doesn't conform to snake_case naming style (invalid-name)
```
Suppose to be showing, but not shown.
when change was on line 91:
```
@bot.command(name="rank", help="Displays the information about the summoner.")
async def get_rank(ctx, name: str):
    try:
        summonerInfo = get_summoner_rank(name)
```

--- a/README.md
+++ b/README.md
@@ -27,7 +27,12 @@ DISCORD_TOKEN={TOKEN}
   ```
 - This will set up auto formatting upon saving a file.
 
-7. To call the bot with the desired prefix locally, 
+7. To enable `pylint` on `pre-push`, please run following command:
+```
+pre-commit install --hook-type pre-push
+```
+
+8. To call the bot with the desired prefix locally, 
 
 - Add `LOCAL_BOT_PREFIX= ""` to .env with desired prefix between the double quotes
 

